SlicerProstate/SlicerProstateUtils/mixins.py

```diff
@@ -246,13 +246,13 @@ def get3DEuclideanDistance(pos1, pos2):
     return distance3D
 
   @staticmethod
-  def dilateMask(label):
+  def dilateMask(label, dilateValue, erodeValue=0):
     imagedata = label.GetImageData()
     dilateErode = vtk.vtkImageDilateErode3D()
     dilateErode.SetInputData(imagedata)
-    dilateErode.SetDilateValue(1.0)
-    dilateErode.SetErodeValue(0.0)
-    dilateErode.SetKernelSize(12, 12, 1)
+    dilateErode.SetDilateValue(dilateValue)
+    dilateErode.SetErodeValue(erodeValue)
+    dilateErode.SetKernelSize(3, 3, 3)
```

Comment:
Note that kernel is specified in voxels, and prostate imaging is very anisotropic (0.5x0.5x4). The original setting, in effect, after multiplying by the voxel spacing, results in a more regular kernel shape.